Sources/Helpers/UIFont+DynamicTypeCustom.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIFont+DynamicTypeCustom.swift
+//  Pods
+//
+//  Created by Nikki Vergracht on 22/05/2017.
+//
+//
+
+import UIKit
+
+/// Available from iOS & tvOS 7.0 upwards.
+@available(iOS 7.0, tvOS 7.0, *)
```

Comment:
The minimum OS version supported by Delirium is **iOS 9.0** and **tvOS 9.0**. So the `@available` clause isn't required.